htdocs/web_portal/views/downtime/edit_downtime.php

```diff
@@ -225,7 +225,7 @@ class="form-control" id="Select_Sites" name="select_sites" size="10"
         $('#startDate').data("DateTimePicker").date("<?php echo date_format($startDate,"d/m/Y"); ?>");
         $('#endDate').data("DateTimePicker").date("<?php echo date_format($endDate,"d/m/Y"); ?>");
 
-        //Set the start and finish times (don't echo in the full date, just the time values, time widget didn�t like timestamp with date)
+        //Set the start and finish times (don't echo in the full date, just the time values, time widget didn�t like timestamp with date)
```

Comment:
??

```suggestion
        //Set the start and finish times (don't echo in the full date, just the time values, time widget didn't like timestamp with date)
```

Reply:
Just to add on. The line you are seeing is only visible in the git(Browser - JS engine seems hard time understanding the characters). But this is not part of the commit.

Reply:
can this change be omitted from this PR?

Reply:
Removed the strange/unidentified character from the PR.

Reply:
ah sorry, I mean't remove the change to this line - so this PR only deals with whitespace/newlines at the end of files. Though please do stick [40942ad](https://github.com/GOCDB/gocdb/pull/457/commits/40942ad73e8089b05c3262974080df1b25da0f0b) in a seperate PR 😄 .

Reply:
@gregcorbett , If you see my initial commit 40e8919cc409e9e8fe1456966e2ca7109bc0dd56. "It doesn't have that change at all".  But GIT or GitHub (Not sure) somehow picked up that line saying their is a strange character.
 Even If I remove that change GIT will complain again regarding that character.

Reply:
If Gits detecting it as a change, you should be able to not commit that change? either way - putting [40942ad](https://github.com/GOCDB/gocdb/pull/457/commits/40942ad73e8089b05c3262974080df1b25da0f0b) into a separate PR will remove the change from this PR.

Reply:
I am attaching the history of "https://github.com/GOCDB/gocdb/commit/40e8919cc409e9e8fe1456966e2ca7109bc0dd56" vs previous commit that showing in red lines of what I have changed.

<img width="853" alt="edit_downtime" src="https://github.com/GOCDB/gocdb/assets/127398882/b34ae8bf-2240-4948-adc0-025d3dc334fe">

Reply:
No @gregcorbett . GIT is NOT showing me that change expect in the browser. I don't see that change when I am doing commit.

Reply:
how weird, it shows up for me in Git as a change. Regardless, putting [40942ad](https://github.com/GOCDB/gocdb/pull/457/commits/40942ad73e8089b05c3262974080df1b25da0f0b) into a separate PR will remove the change from this PR.

Reply:
Deleted the change from this PR @gregcorbett 👍

Reply:
This is being addressed as part of https://github.com/GOCDB/gocdb/pull/471.